libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/helpers/VcHelper.java

```diff
@@ -65,6 +65,15 @@ public static boolean isSuccessfulVc(SignedJWT vc) throws ParseException {
         return isValidEvidence(credentialEvidenceList);
     }
 
+    public static boolean isSuccessfulVcs(List<SignedJWT> vcs) throws ParseException {
```

Comment:
`areSuccessfulVcs`?

I hate that you can't do this with a simple stream because Java :'(